extension/src/pipeline/index.ts

```diff
@@ -150,6 +152,16 @@ export class Pipeline extends DeferredDisposable {
   }
 
   private async addPipeline() {
+    const response = await Modal.showInformation(
```

Comment:
[I] Actually need to move things around or bypass this for plots

Reply:
I think the best thing to do would be break up `getCwd` (which checks for the stage) and move the check upfront for adding plots, along with a modal that tells the user what is going on ... come back to it on Monday